react-form-controls/my-app/src/RegistrationFormUncontrolled.js

```diff
@@ -0,0 +1,24 @@
+
+export default function RegistrationFormUncontrolled({text}) {
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    const formJson = Object.fromEntries(formData.entries());
+    console.log('state:',formJson);
+  }
+
+  return (
+    <form method="" onSubmit={ handleSubmit }>
+      <label>
+        Username: <input name="username" type="text" required={true}/>
```

Comment:
A nice little shortcut with boolean props is that they default to true, so you can just say:
`<input name="username" type="text" required />`
But to set them to `false` you have to use `{false}`.